cogs/tags.py

```diff
@@ -161,10 +164,14 @@ async def list(self, ctx, *, member: discord.Member = None):
     @tag.command()
     @commands.check_any(checks.is_moderator(), commands.has_role("Create Tags"))
     async def create(self, ctx, name, *, content):
-        """Creates a new tag owned by you."""
+        """Creates a new tag owned by you. Attach images/gifs/videos to automatically append their urls to the tag."""
 
-        if len(content) > 1994:
-            await ctx.send("Tag content must be at most 1994 characters.")
+        for attachment in ctx.message.attachments:
+            if attachment.content_type.split("/")[0] in ("image", "video"):
```

Comment:
not sure if checking image/video is necessary, attaching other files might be ok too

Reply:
Good point. Once concern I have tho is that when you open an attachment url it directly downloads to your device (unless you have it set to prompt each time, which is not the default in some cases) so it may be dangerous in some cases. May just be overthinking, but I'm not sure if other attachments are really necessary. Your call, what do you say?

Reply:
i think it's ok. someone could always attach a link to a bad file anyway

Reply:
Alright 👍

Reply:
Done!